app/models/status/errored.rb

```diff
@@ -0,0 +1,7 @@
+module Status::Errored
```

Comment:
I would like to difference this from `Failed`, because this is much more severe: i.e. your program doesn't even compile. What do you think?